routes/userRouter.js

```diff
@@ -171,13 +171,20 @@ router.get('/recently-viewed-artists', async (req, res) => {
 });
 
 router.post('/recently-viewed-artists', async (req, res) => {
-  const { userId, artistId } = req.body;
+  const userId = req.query.userId;
+  const artistId = req.query.artistId;
```

Comment:
standard practice is to deconstruct request variables
const {userId, artistId} = req.query;